.github/workflows/brakeman.yml

```diff
@@ -16,4 +16,4 @@ jobs:
         gem install --no-format-executable brakeman -v 5.4.0
     - name: Brakeman
       run: |
-        brakeman --rails6 -p src/api
+        brakeman -p src/api
```

Comment:
Shouldn't we force to Rails 7, so it doesn't use a 7+ version before we adapt our code?

Reply:
There's no Rails 7 mode, see this screenshot from Brakeman's help:

![no-rails-7-mode](https://user-images.githubusercontent.com/1102934/208632451-e45fdd90-8134-47a0-ace9-e330f4f712a1.png)